RELEASE.md

```diff
@@ -0,0 +1,160 @@
+
+_Outcome_: __Release branch has `readme.txt` is updated with release details.__
+
+#### Build zip & smoke test
+- Ensure you are on the tip of the release branch, e.g. `git pull origin release/2.5`
+- Update dependencies – `npm ci`.
+- Run a production build - `npm run build`.
+- Run package script to get a zip to test `npm run package-plugin`.
+- Smoke test built release zip:
+  - __Recommended__: 
+    - At least one other person should test the built zip - ask a teammate to help out.
+    - Test in a clean environment, e.g. Jurassic.Ninja site.
+  - Confidence check - check blocks are available and function.
+  - Test to confirm new features/fixes are working correctly.
+  - Smoke test – test a cross section of core functionality.
+
+_Outcome_: __Confident that source code is ready for release: intended fixes are working correctly, no release blockers or build issues.__
+
+### Release!
+#### Tag release on GitHub
+- Prepare tagged release on github `npm run deploy`. 
+  - Note: the script automatically updates version numbers (commits on your behalf).
+- Edit release, add changelog info to Github release notes.
+- Check release repo tag is correct - checkout, smoke test/confidence check.
+
+_Outcomes_: __Version numbers updated in source code & developers can test tagged release.__
+
+#### Release to WPORG
+- Run `npm run release`. This script clones a copy of the source code to your home folder, and outputs an `svn` command to push release up to WPORG.
+- Push release to WPORG using `svn`.
+  - Run generated svn command to commit to WPORG svn repo.
+    - The command should look like this: `cd /Users/{YOU}/blocks-deployment/woo-gutenberg-products-block-svn && svn ci -m "Release 2.5.11, see readme.txt for changelog."`
+  - Commit should complete successfully with a message like `Committed revision 2231217.`.
+- Confirm that the WPORG release is updated and correct:
+  - Changelog, `Version` & `Last updated` on [WPORG plugin page](https://wordpress.org/plugins/woo-gutenberg-products-block/).
+  - Confirm svn tag is correct, e.g. [2.5.11](https://plugins.svn.wordpress.org/woo-gutenberg-products-block/tags/2.5.11/).
+  - Confirm [WooCommerce.com plugin page](https://woocommerce.com/products/woocommerce-gutenberg-products-block/) is updated.
+  - Download zip and smoke test.
+  - Test updating plugin from previous version.
+    - Recommended: test existing WooCommerce Blocks content works correctly after update (no block validation errors).
+
+_Outcome_: __Customers can install/update via WPORG; WPORG plugin page is up to date__.
+
+### After release
+#### Update `master` with release changes
+- Ensure changelog is up to date on master.
+- For _major_ & _minor_ releases, update version on master with dev suffix, e.g. [`2.6-dev`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/e27f053e7be0bf7c1d376f5bdb9d9999190ce158).
+- Code changes should already be present on master - all dev happens on `master`.
```

Comment:
I'm not sure to understand this line. Which code changes is it referring to?

Reply:
I think this refers to ongoing development since we merge all changes to master first.  This line can probably be removed.